src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java

```
@@ -0,0 +1,146 @@
+    {
+        @Override
+        protected Optional<String> parseResult(final String line)
+        {
+            return Optional.of(line);
+        }
+    };
+
+    /**
+     * Allows you to check if you have installed tippecanoe on your system and if it satisfies the
+     * minimum version that we need.
+     *
+     * @return Whether you have a valid tippecanoe ready to execute in your environment.
+     */
+    public static boolean hasValidTippecanoe()
+    {
+        final String[] commandArray = new String[] { "tippecanoe", "--version" };
+        try
+        {
+            RunScript.run(commandArray, Collections.singletonList(MONITOR));
+        }
+        // When you look up the version, tippecanoe exits with 1, so getting here is normal.
+        // We want to get into this catch.
+        catch (final CoreException exception)
+        {
+            final Optional<String> result = MONITOR.getResult();
+            if (result.isPresent())
+            {
+                final String outputString = result.get();
+                final String[] versionArray = outputString.split("\n")[0].split("tippecanoe v");
+                // Here we extract the version.
+                if (versionArray.length == 2)
+                {
+                    final String versionString = versionArray[1];
+                    final DefaultArtifactVersion version = new DefaultArtifactVersion(
+                            versionString);
+                    if (TippecanoeSettings.MIN_VERSION.compareTo(version) <= 0)
+                    {
+                        return true;
+                    }
+                    else
+                    {
+                        logger.error(
+                                "Your version of tippecanoe is too old! The minimum version is {}",
+                                TippecanoeSettings.MIN_VERSION);
+                    }
+                }
+            }
+        }
```

It might be worth adding some sort of "unexpected tippecanoe return value" or "failed to parse" log messages here and inside the `catch` block at the end.

That way if `tippecanoe` is ever patched to fix the `--version` exit code, or if the format of `--version` changes you won't just get a silent failure.